internal/command/admin/deploy/docker/docker_up.go

```diff
@@ -35,8 +35,21 @@ const (
 )
 
 type Up struct {
-	CoreImage   string `name:"core" short:"c" help:"core docker image to deploy" default:"${default_core_image}"`
-	WorkerImage string `name:"worker" short:"w" help:"worker docker image to deploy" default:"${default_worker_image}"`
+	CoreImage   string `name:"core" short:"c" help:"Core docker image to deploy" default:"${default_core_image}"`
+	WorkerImage string `name:"worker" short:"w" help:"Worker docker image to deploy" default:"${default_worker_image}"`
+}
+
+func (f *Up) Help() string {
+	return `
+DESCRIPTION
+
+	It creates a local Docker-based Funless deployment.
```

Comment:
Since we picked up the FunLess variant of the name (with the big L) you should update the name wherever you used Funless to FunLess.